src/Tribe/Admin/Notice/Plugin_Download.php

```diff
@@ -96,10 +106,14 @@ public function show_inactive_plugins_alert() {
 			'a'      => array( 'href' => array() ),
 		);
 
+		$pue_notice_html = '<p>' . esc_html__( '---MESSAGE_PLACEHOLDER---', 'tribe-common' ) . '</p>';
```

Comment:
What use is the translation aspect here?

Reply:
I am waiting for Leah's message here. That is why I left a place holder there. =)

Reply:
Gotcha, thanks! I put it on hold but code looks good to me!